src/applications/vaos/tests/appointment-list/components/PastAppointmentsList/index.unit.spec.jsx

```diff
@@ -404,4 +404,96 @@ describe('VAOS integration: past appointments', () => {
       expect(ranges[5].endDate).to.include('2019-12-31T23:59:59');
     });
   });
+
+  it('should include video appointments with a FUTURE status', async () => {
+    const appointment = getVideoAppointmentMock();
+    appointment.attributes = {
+      ...appointment.attributes,
+      facilityId: '983',
+      clinicId: null,
+      startDate: pastDate.format(),
+    };
+    appointment.attributes.vvsAppointments[0] = {
+      ...appointment.attributes.vvsAppointments[0],
+      dateTime: pastDate.format(),
+      bookingNotes: 'Bring face mask',
+      status: { description: 'F', code: 'FUTURE' },
+    };
+    mockPastAppointmentInfo({ va: [appointment] });
+
+    const screen = renderWithStoreAndRouter(<PastAppointmentsList />, {
+      initialState,
+    });
+
+    await screen.findByText(
+      (_, node) => node.textContent === 'VA Video Connect at home',
+    );
+
+    expect(screen.queryByText(/You don’t have any appointments/i)).not.to.exist;
+  });
+
+  it('should not show va appointments with an excluded status', async () => {
```

Comment:
What flag is used to determine 'excluded' status?

Reply:
There are a set of statuses that we exclude from being displayed: https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/vaos/services/appointment/index.js#L71